packages/@aws-cdk/aws-ecs/test/integ.scheduled-ecs-task.ts

```diff
@@ -0,0 +1,34 @@
+import ec2 = require('@aws-cdk/aws-ec2');
+import cdk = require('@aws-cdk/cdk');
+import ecs = require('../lib');
+
+import path = require('path');
+import { ScheduledEc2Task } from '../lib';
+
+const app = new cdk.App();
+
+class ScheduledTaskStack extends cdk.Stack {
```

Comment:
Do we need to wrap the stack in a class? It's a little confusing to have a wrapper for the construct that we are actualy testing -- not sure if there's a precedent for this elsewhere?

Reply:
I was following the existing integration test structure here: https://github.com/awslabs/aws-cdk/blob/master/packages/%40aws-cdk/aws-ecs/test/ec2/integ.event-task.lit.ts

Reply:
Ah, I see this construct was based off of that. It might be worth just replacing what's in the existing test to use the new L3. Thoughts, @rix0rrr ?

Reply:
Do we think it's worth having a separate test still, or can we replace the existing one?

Reply:
There's a case for either. Keeping it ensures that in the future any changes to the scheduledEc2Task construct that could be additional functionality that was different from what's tested in integ.event-task.js. That being said, as of this pr, the tests behave identically, so I'm okay either way.

Reply:
Let's combine them.